README.md

```diff
@@ -3,13 +3,17 @@
 [![Greenkeeper badge](https://badges.greenkeeper.io/hmcts/nodejs-logging.svg)](https://greenkeeper.io/)
 
 A logging component used by Reform's Node.js applications. Some background info:
```

Comment:
I would recommend adding header note (in bold) saying that this is not compatible with reform tactical logging spec and that for tactical applications logger 2.x should be used.